readthedocs/templates/builds/build_detail.html

```diff
@@ -76,7 +76,7 @@
               style="display: none;">
           {{ build.get_state_display }}
         </span>
-        <img src="{{ MEDIA_URL }}images/loader.gif"
```

Comment:
Might be worth doing a sweep for MEDIAA_URL in our templates at some point, if we can replace all usage of it with the {% static %} tag.